src/components/Discover/DiscoverHeader/PeriodSelector.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Select, MenuItem } from '@material-ui/core';
+
+const periods = [
+  { label: 'hour', query: '1h' },
+  { label: 'day', query: '1d' },
+  { label: 'week', query: '1w' },
+  { label: '2 weeks', query: '2w' },
+  { label: '30 days', query: '30d' },
+  { label: '90 days', query: '90d' },
+  { label: 'year', query: '1y' },
+];
+
+interface IProps {
+  period: string;
+  setPeriod: (period: string) => void;
+}
+
+function PeriodSelector(props: IProps): React.ReactElement {
+  const { period, setPeriod } = props;
+
+  const handleChange = (event: any) => {
```

Comment:
이벤트 타입을 지정해주시면 좋을것 같습니다!
```suggestion
  const handleChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
```
타입이 맞는지는 테스트해봐야 알것같네요...

Reply:
HTMLInputElement 넣다가 에러가 나서 빼놨는데 SelectElement가 따로 있었군요... 감사합니다!! 바로 고치겠습니다.

Reply:
고쳐도 타입 에러가 나네요... Material의 Select가 진짜 Select 엘리먼트가 아니라서 충돌이 난다고 합니다. 검색해보니 라이브러리에 관련 이슈가 아직 있는 것 같아서 일단 any로 남겨두도록 하겠습니다.